sndev

```diff
@@ -305,6 +333,15 @@ sndev__help_stacker_lncli() {
   docker__stacker_lnd --help
 }
 
+sndev__stacker_lightning-cli() {
```

Comment:
This isn't a valid identifier in posix shell because of the `-` and the script fails to load for me.

I've been running my script changes through https://www.shellcheck.net/ just in case and heeding important warnings.

Reply:
I fixed this locally already